parser/test/test_oob.h

```diff
@@ -18,4 +18,6 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301,USA
  */
 
-void test_oob(const str *, void (*)(const str *, void *), void *);
+enum oob_position {OOB_PRE, OOB_POST};
```

Comment:
I'm not a big fan of this enum, as its purpose is not immediately obvious.  Would you also agree that these alternatives would be net superior, or is it just me?

* `enum oob_test {OOB_UNDERFLOW, OOB_OVERFLOW}`
* `enum oob_test {OOB_UNDERRUN, OOB_OVERRUN}`